site/content/contribute/server/developer-setup/windows.md

```diff
@@ -18,11 +18,11 @@
     If you don't migrate your data, the new, docker-compose-managed containers will start out empty. To remove the old containers -- destroying any existing data -- use `make clean-old-docker`.
      
     
-3. Download and install Go from https://golang.org/dl/
+3. Download and install Go from https://golang.org/dl/ . Follow the getting started steps to define your workspace directory and %GOPATH%.
```

Comment:
Since go 1.12 and the use of modules in the server, cloning the repo into your `GOPATH` is not longer needed.

Reply:
Hi, I am not clear on what the result is of your comment. So does that mean that _'Step 4. Clone the Mattermost source code from your fork'_ in the guidance isn't relevant anymore?

Reply:
Sorry for being not clear here. This means:
- You don't longer need a defined `GOPATH`
- Step 4 should just be:
```
git clone https://github.com/{yourgithubusername}/mattermost-server.git
cd mattermost-server
git config core.eol lf
git config core.autocrlf input
git reset --hard HEAD
```

Reply:
So I don't think my PR is needed anymore, but step 4 where we define the GOPATH and Step 6 need to be updated? Are you able to propose those changes and review end to end?

Reply:
I've opened an issue for this: https://github.com/mattermost/mattermost-developer-documentation/issues/380

Reply:
Awesome, thanks.